.gitignore

```diff
@@ -54,6 +54,7 @@ docs/html
 
 # Python venv
 .venv
+cirque_venv
```

Comment:
adding more top_level directories is somewhat inconvenient. Could we use `out` for example? we do so for default python builds.